core/src/test/scala/org/apache/spark/scheduler/OutputCommitCoordinatorSuite.scala

```diff
@@ -235,7 +226,6 @@ class OutputCommitCoordinatorSuite extends SparkFunSuite with BeforeAndAfter {
     assert(!outputCommitCoordinator.canCommit(stage, 3, partition, taskAttempt))
     outputCommitCoordinator.taskCompleted(stage, 1, partition, taskAttempt,
       ExecutorLostFailure("0", exitCausedByApp = true, None))
-    assert(outputCommitCoordinator.canCommit(stage, 4, partition, taskAttempt))
```

Comment:
we need to update the code comment before this code chunk.

Reply:
How about current?

Reply:
It has not been updated yet
```
    // Commit the 1st attempt, fail the 2nd attempt, make sure 3rd attempt cannot commit,
    // then fail the 1st attempt and make sure the 4th one can commit again.
```

Reply:
Updated